.github/workflows/wheels.yml

```diff
@@ -1,11 +1,22 @@
 # Workflow to build and test wheels
+# =================================
+# This github action gets triggered whenever there
+# is a push to the master branch. It generates both
+# wheels and distributions files, making sure their
+# contents are correct via unit-testing.
+#
+# However, artifacts are only uploaded to a release
+# here: https://github.com/automl/ConfigSpace/releases
+# When a release tag is created. For example, if
```

Comment:
Hey, could you please be more specific here? In particular, does it have to be a release (I think that's github nomenclature) or a tag (that's git nomenclature)? I think it would also be good if you could briefly add the steps that are required to be done from the command line.